src/App.js

```diff
@@ -23,29 +23,31 @@ export default function App() {
   const classes = useStyles();
 
   const [shouldRunSUSHI, setShouldRunSUSHI] = useState(false);
-  const [text, setText] = useState('Edit FSH Here!');
+  const [inputText, setInputText] = useState('Edit FSH Here!');
+  const [outputText, setOutputText] = useState('Your JSON Output Will Display Here: ');
 
-  function updateShouldRunSUSHI(shouldRunSUSHI) {
+  function buttonFunctionality(shouldRunSUSHI, sushiOutput) {
```

Comment:
I'd suggest using a more descriptive name... This is the Run button, right?  Maybe `handleRunButton` or `onRunButtonClick`?